cmd/metro/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/razorpay/metro/internal/boot"
+	"github.com/razorpay/metro/service/producer"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	// Initialize context
+	ctx, cancel := context.WithCancel(boot.NewContext(context.Background()))
+	defer cancel()
+
+	// Init app dependencies
+	env := boot.GetEnv()
+	err := boot.InitProducer(ctx, env)
```

Comment:
@Abhishekvrshny App flavor(server, consumer, producer) based boot-up changes to follow?

Reply:
@shivasishdas yes, would do that next.

Reply:
Cool!